styles.css

```diff
@@ -242,3 +242,98 @@ nav {
   width: 40px;
   margin-right: 20px;
 }
+
+.contact-form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(assets/backgroundform.png);
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  background-size: 61%;
+  background-position-x: right;
+  background-position-y: 20px;
+  border-radius: 90px 0 0 0;
+  margin-top: 103px;
+}
+
+.contact-title {
+  text-align: center;
+  font-size: 40px;
+  color: white;
+  margin-bottom: 20px;
+  padding-top: 103px;
+}
+
+.p-contact {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  padding: 24px;
+}
+
+.contact-info {
+  display: flex;
+  flex-direction: column;
+  padding: 20px 5px 5px 5px;
+}
+
+.inp-form {
+  width: 100%;
+  height: 43px;
+  margin-bottom: 20px;
+  border-radius: 8px;
+  font-size: 17px;
+  border: none;
+  color: #172b4d;
+  padding: 10px 15px;
+}
+
+.hidden {
+  display: none;
+}
+
+.text-area {
+  width: 100%;
+  height: 160px;
+  font-size: 17px;
+  color: #172b4d;
+  margin-bottom: 20px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 10px 15px;
+}
+
+.contact-bar {
+  text-align: center;
+}
+
+.btncontact {
```

Comment:
- [OPTIONAL] I think it's better to create a style for the button when pressed as in the Figma template. It's a project requirement to stick to the template. As not required by the previous reviewer I consider that optional.